lte/gateway/python/scripts/dhcp_helper_cli.py

```diff
@@ -107,7 +112,7 @@ def allocate(self):
 
     def release(self):
         self.send_dhcp_release()
-        self.wait_for(self.receive_dhcp_release)
+        # Receiving release ack is not mandatory
```

Comment:
Does this comment describe why `self.wait_for(self.receive_dhcp_release)` was removed? I.e., does this comment only make sense if someone is viewing the diff? Then I would remove it.

Reply:
I would say it describes why we don't wait for receiving an ack response as compared to the methods `allocate` and `renew`